src/routes/docs/tutorials/pagination/step-1/+page.markdoc

```diff
@@ -0,0 +1,30 @@
+---
+layout: tutorial
+title: Build a todos app with react
+description: Learn about using pagination in a react app using appwrite backend.
+step: 1
+difficulty: beginner
+readtime: 10
```

Comment:
This is a lie, not 10 minutes

Reply:
should the difficulty  for the tutorial be changed from `beginner` to `Intermediate`?

Reply:
Yeah intermediate

Reply:
20 min readtime, maybe?